### PR TITLE
[Fastboot] Delay LLDP service for better fastboot performance

### PR DESCRIPTION
**Pull request by shlomibitton**
Signed-off-by: Shlomi Bitton <shlomibi@nvidia.com>

#### Why I did it

Profiling the system state on init after fast-reboot during create_switch function execution, it is possible to see few python scripts running at the same time.
This parallel execution consume CPU time and the duration of create_switch is longer than it should be.
Following this finding, and the motivation to ensure these services will not interfere in the future, LLDP is delayed in 90 seconds until the system finish the init flow after fastboot.

bootchart_no_optimizations

#### How I did it

- Add a timer for LLDP service.
- Copy the timer file to the host bin image.

#### How to verify it

Run fast-reboot on MLNX platform and observe faster create_switch execution time.
This PR is dependent on PR: https://github.com/Azure/sonic-buildimage/pull/10567
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012
- [ ] 202106
- [x] 202111

### CODE REVIEW

**Review by yxieca — APPROVED** [review]

File: files/build_templates/per_namespace/lldp.timer
Comment (stepanblyschak) [inline files/build_templates/per_namespace/lldp.timer:9]: lldp.service is a per-namespace container. Will that work for multi-asic? Should we have several timers?

**Comment by vaibhavhd** [comment]
I think this change will impact all kinds of reboot - warm/fast/cold/etc. Do you agree?

If yes, is that what we want to do? Can we do this only for fast and warmboot case?

**Comment by shlomibitton** [comment]
> I think this change will impact all kinds of reboot - warm/fast/cold/etc. Do you agree?
> 
> If yes, is that what we want to do? Can we do this only for fast and warmboot case?

@vaibhavhd I agree, I added a condition for it on syncd.sh like we do for pmon today.

File: files/scripts/syncd.sh
Comment (vaibhavhd) [inline files/scripts/syncd.sh:53]: Minor: This is correct, but it may be better to match against whitelist BOOT_TYPEs (fast, warm, fastfast).
This is so that this condition does not evaluate to true for any other boottype that gets added in future.

File: files/build_templates/per_namespace/lldp.timer
Comment (vaibhavhd) [inline files/build_templates/per_namespace/lldp.timer:2]: Correction - fast-reboot and warm-reboot performance.

**Review by vaibhavhd — DISMISSED** [review]
Minor comments pending, LGTM otherwise. Please address Stepan's concerns before merging.

File: files/build_templates/per_namespace/lldp.timer
Comment (stepanblyschak) [reply files/build_templates/per_namespace/lldp.timer:9]: Add  has_timer to this file https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/init_cfg.json.j2#L32

**Comment by shlomibitton** [comment]
@vaibhavhd @stepanblyschak I fixed your comments, can you please review and approve?
Thanks

File: files/build_templates/per_namespace/lldp.timer
Comment (shlomibitton) [reply files/build_templates/per_namespace/lldp.timer]: Done

File: files/scripts/syncd.sh
Comment (shlomibitton) [reply files/scripts/syncd.sh]: Done

File: files/build_templates/per_namespace/lldp.timer
Comment (shlomibitton) [reply files/build_templates/per_namespace/lldp.timer]: Done

**Comment by liat-grozovik** [comment]
@vaibhavhd @stepanblyschak kindly reminder to review

**Review by vaibhavhd — APPROVED** [review]

**Review by stepanblyschak — APPROVED** [review]

**Comment by qiluo-msft** [comment]
This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

**Comment by shlomibitton** [comment]
@qiluo-msft I created a separate PR for 202012:
https://github.com/Azure/sonic-buildimage/pull/10744

**Pull request merged**
